syntax/diff.sublime-syntax

```diff
@@ -0,0 +1,62 @@
+%YAML 1.2
+---
+# http://www.sublimetext.com/docs/3/syntax.html
+name: GitSavvy Diff
+hidden: true
+scope: git-savvy.diff
+first_line_match: |-
+  (?x)^
+      (===\ modified\ file
+      |==== \s* // .+ \s - \s .+ \s+ ====
+      |Index:[ ]
+      |---\ [^%]
+      |\*\*\*.*\d{4}\s*$
+      |\d+(,\d+)* (a|d|c) \d+(,\d+)* $
+      |diff\ --git[ ]
+      )
+contexts:
+  main:
+    - match: '(^(-{3}) a/(.+)$\n?)'
+      comment: Needs to be first.
```

Comment:
This comment should be improved or removed.  It doesn't tell me why it needs to be first.